api/router/pubsub/CiEventHandler.go

```diff
@@ -80,6 +80,7 @@ func (impl *CiEventHandlerImpl) Subscribe() error {
 		if err != nil {
 			return
 		}
+
```

Comment:
change branch name

Reply:
also add description, github issue link, and test cases like how have you tested this fix